### DISCUSSION TITLE
Cool images of Earth

### DISCUSSION THREAD

**Discussion by thku8507** [earthlab-education/Earth-Analytics-2023-01-Intro#127] [Show and tell]
This isn't really something I "made", but over the course of hundreds (thousands?) of hours procrastinating by clicking around on Google Earth I've come across some beautiful landscapes. I've saved them as high-res images free of clutter and put them in a public google drive folder [linked here](https://drive.google.com/drive/folders/1UgM-Jj-ZXNXS67HTzzjqYAEA0hJxv5cr?usp=sharing), so feel free to use them for anything--PowerPoints, syllabi, etc. 

![panama - Copy](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/143355157/dc812a6f-bdab-41be-946c-6ff587cd606d)

**Comment by ellenalamont17** [comment]
Actually that is a really great idea! I can't tell you the number of times I have come across awesome landscapes in google earth and though that having that image would be nice. Never really crossed my mind to actually save them! Thanks for sharing!

**Reply by thku8507** [reply]
Feel free to add any you come across to make it a collaborative folder! I think the permissions in there should allow that.
